api/admin/coordinator.py

```diff
@@ -53,8 +53,21 @@ class BaseCoordinatorGroupInline(
     max_num = 0
 
 
+class NameGroupChoiceField(forms.ModelChoiceField):
+
+    def label_from_instance(self, obj: Any) -> str:
```

Comment:
is `Group`'s `__str__` method not good enough?

Reply:
Then we will change it for everything, do we want it?

Reply:
But I think `Group.__str__()` method already gives [decent enough information](https://github.com/samanthasgroup/django-webapps/blob/master/api/models/group.py#L119-L125) as it is, does it not?

Reply:
![image](https://github.com/samanthasgroup/django-webapps/assets/59690441/89869fd3-e334-4045-b43f-61ef498fc77b)

Reply:
> ![image](https://private-user-images.githubusercontent.com/59690441/319781622-89869fd3-e334-4045-b43f-61ef498fc77b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTIyNjc1NTEsIm5iZiI6MTcxMjI2NzI1MSwicGF0aCI6Ii81OTY5MDQ0MS8zMTk3ODE2MjItODk4NjlmZDMtZTMzNC00MDQ1LWI0M2YtNjFlZjQ5OGZjNzdiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDA0VDIxNDczMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI5MzY3Nzk4M2ZjMTcwOGVhYWNjZjM0ZWE5MDQzZjYyODBjM2RlMTAwZTY3OGRkZWUwMzU3ZjIyMjIwN2I2MjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.uOzsmI4qXwFWw3Uy0VlpfO1eICC0eUwhTGuea28EFAI)

Oh, right, things get erased from my memory. They requested it to be less verbose. Well, then "2: " is coordinator's ID :) So it should be something like
```py
",".join([f"{coordinator.id}: {coordinator.name}" for coordinator in obj.coordinators])
```

Reply:
Did it in https://github.com/samanthasgroup/django-webapps/pull/483/commits/92c2a8e77dde651d269770e1a66547b57f6a7192